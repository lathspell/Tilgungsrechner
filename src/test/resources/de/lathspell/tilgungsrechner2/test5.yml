months:
    "2016-04":
        darlehensbetrag: 1000
        sollzins: -10.00
        monatsrate: 100
        auszahlung: 1000
    