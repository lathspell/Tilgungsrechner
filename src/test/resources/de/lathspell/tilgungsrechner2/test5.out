# test5
Tilgungsplan
Periode 	Rate (EUR) 	Zinsanteil (EUR) 	Tilgungsanteil (EUR) 	Restschuld am Periodenende (EUR)
Apr 2016           100,00              -8,33                 108,33                    891,67
Mai 2016           100,00              -7,43                 107,43                    784,24
Jun 2016           100,00              -6,54                 106,54                    677,70
Jul 2016           100,00              -5,65                 105,65                    572,05
Aug 2016           100,00              -4,77                 104,77                    467,28
Sep 2016           100,00              -3,89                 103,89                    363,39
Okt 2016           100,00              -3,03                 103,03                    260,36
Nov 2016           100,00              -2,17                 102,17                    158,19
Dez 2016           100,00              -1,32                 101,32                     56,87
Jan 2017            56,40              -0,47                  56,87                      0,00
